kaldb/src/test/java/com/slack/kaldb/chunk/ChunkCleanerTaskTest.java

```diff
@@ -30,13 +33,15 @@
 
   @Before
   public void setUp() throws Exception {
+    KaldbConfigUtil.initEmptyIndexerConfig();
```

Comment:
This was removed in #110, but the test currently still requires this config to be initialized. 

It initially passed tests with this removed because another test that didn't cleanup its config was running before this one. With this refactor the tests are running in a slightly different order, causing this test to now fail.